### PR TITLE
Align hero title

### LINKED ISSUE #42: Align the About us text with the hero banner title
image

### PR DESCRIPTION
**Pull request by JiangLong2019**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #42 

Test URLs:
- Original: https://www.sunstar-foundation.org/en/about-us/senri-clinic
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/en/about-us/senri-clinic
- After: https://issue-42--sunstar-foundation--hlxsites.hlx.live/en/about-us/senri-clinic

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------

- [ ] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------

### CODE REVIEW

**Comment by JiangLong2019** [comment]
Hi @ehrro, this has been fixed, please help to check.

image

**Review by dnbute — APPROVED** [review]
LGTM

File: blocks/hero-banner/hero-banner.css [hunk 1]
```diff
@@ -242,16 +239,31 @@
...
+        right: 100%;
+    }    
+
     .hero-banner .hero-banner-button-container span {
         width: 100%;
         font-size: var(--heading-font-size-xxs);
         margin-left: unset;
     }
 
+    .hero-banner .hero-banner-heading-section{
```
Comment (sdmcraft) [inline blocks/hero-banner/hero-banner.css:258]: indentation is incorrect.

File: blocks/hero-banner/hero-banner.css [hunk 2]
```diff
@@ -208,14 +207,12 @@
...
     .hero-banner .hero-banner-text-container {
-        max-width: calc(55vw);
-        width: 63.75rem;
-        padding: unset;
-    }
-
-    .hero-banner .hero-banner-text-wrapper {
-        margin-bottom: unset;
-        padding: 3rem;
+        display: block;
```
Comment (sdmcraft) [inline blocks/hero-banner/hero-banner.css:210]: Why is `display: block;` needed?

Comment (JiangLong2019) [reply blocks/hero-banner/hero-banner.css]: Fixed

**Pull request merged**